src/docs/asciidoc/chapters/chapter3.adoc

```diff
@@ -471,8 +471,8 @@ Bootstrap also includes a number of utility classes that can be used to show and
 
 Bootstrap's classes for setting the display are names using the following format:
 
-* `.d-{value}` for `xs`
-* `.d-{breakpoint}-{value}` for `sm`, `md`, `lg`, and `xl`
+* `.d-\{value}` for `xs`
+* `.d-\{breakpoint}-\{value}` for `sm`, `md`, `lg`, and `xl`
```

Comment:
Do I always have to escape curly braces from now on? I'd prefer it if the AsciiDoc I write doesn't have to change.

Reply:
It was already an issue but a silent one. Antora is using `attribute-missing=warn`, that's how I found them.

In this case Asciidoctor will try to resolve `{breakpoint}` and `{value}`, fortunately you didn't define these attributes so they won't be replaced... but it's "unsafe".
If you don't like this syntax, we can alternatively use:

```
`+.d-{breakpoint}-{value}+`
```